### PR TITLE
Rolled up updates

### PR DESCRIPTION
**Pull request by jhutchings1**
👋🏻  @SharonManyara I went ahead and took all of your pull requests from the past few days and I retargeted them against this new branch called `rolled-up-updates` using this technique. This will let us review all of the changes together on staging before putting them out on the production site. You can do this too if you are working on something and accidentally send a PR to the default branch before it's ready. Here are some instructions for you :) https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request

### CODE REVIEW

**Comment by jhutchings1** [comment]
Also, it looks like your changes for the how to pay your dues document didn't have the same filename, so I'll have to go and change the link on that.

**Comment by jhutchings1** [comment]
@SharonManyara Do you mind clicking around the staging site to see if everything looks like you expect it to? https://klahanie.github.io/klahanie

**Review by SharonManyara — CHANGES_REQUESTED** [review]
@jhutchings1 
i just realized the application says 2020 instead of 2021. This is the updated file 
[Aquatics Lifeguard Employment Application 2021.pdf](https://github.com/Klahanie/klahanie/files/5852141/Aquatics.Lifeguard.Employment.Application.2021.pdf)

**Comment by jhutchings1** [comment]
@SharonManyara Do you want to switch to the rolled-up-updates branch and make the change by uploading the new file? Don't want to cut off a learning opportunity.

File: _amenities/pools/index.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ show-title: true
 * [Mountainview Pool Schedule]({{site.url}}/amenities/pools/mountainview-pool.html)
 * [Swim Lessons]({{site.url}}/amenities/pools/swim-lessons.html)
 * [Pool rules]({{site.url}}/amenities/pools/pool-rules.html)
+* [Lifeguard Employment Application]({{site.url}}/files/Aquatics Lifeguard Employment Application 2020.pdf)
```
Comment (SharonManyara) [inline _amenities/pools/index.md]: [Aquatics Lifeguard Employment Application 2021.pdf](https://github.com/Klahanie/klahanie/files/5913117/Aquatics.Lifeguard.Employment.Application.2021.pdf)

Comment (SharonManyara) [reply _amenities/pools/index.md]: updating new file

**Review by SharonManyara — CHANGES_REQUESTED** [review]

File: _amenities/pools/index.md (hunk 1 above)
Comment (SharonManyara) [reply _amenities/pools/index.md]: @sspicer1

**Comment by SharonManyara** [comment]
@jhutchings1 how do i correct exit code 1?

File: _amenities/pools/Lifeguard Employment Application [hunk 2]
```diff
@@ -0,0 +1 @@
+({{site.url}}/files/Aquatics Lifeguard Employment Application 2021.pdf)
```
Comment (jhutchings1) [inline _amenities/pools/Lifeguard Employment Application]: This file shouldn't exist. Delete it?

**Comment by jhutchings1** [comment]
> @jhutchings1 how do i correct exit code 1?

This is your problem, the file you are linking to doesn't exist. Double check the name of the file and confirm that it's the same in both places. 
- ./_site/amenities/pools/index.html
  *  internally linking to /files/Aquatics Lifeguard Employment Application 2020.pdf, which does not exist (line 0)
     <a href="/files/Aquatics Lifeguard Employment Application 2020.pdf">Lifeguard Employment Application</a>

**Review by sspicer1 — APPROVED** [review]

**Review by sspicer1 — CHANGES_REQUESTED** [review]
looks like there is a conflict line 13-17

**Review by SharonManyara — CHANGES_REQUESTED** [review]
@sspicer1 i corrected line 13-17 (i hope). Please double check staging site (Thank you!)

**Review by SharonManyara — APPROVED** [review]

**Review by sspicer1 — APPROVED** [review]

**Review by sspicer1 — APPROVED** [review]

**Comment by jhutchings1** [comment]
I went ahead and cleared up the merge conflicts and fixed the broken links that were getting flagged in the tests. Merging.

**Pull request merged**